examples/atlas-search-deployment/README.md

```diff
@@ -0,0 +1,11 @@
+# MongoDB Atlas Provider - Atlas Cluster with dedicated Search Nodes Deployment
+
+This example shows how you can use Atlas Dedicated Search Nodes in Terraform. As a prerequisite, you need to create a project and cluster resource.
```

Comment:
this feels like the prerequisite has to be done separately instead it's part of the example
```suggestion
This example shows how you can use Atlas Dedicated Search Nodes in Terraform. As part of it, a project and cluster resource are created as a prerequisite.
```